wled00/mqtt.cpp

```diff
@@ -166,6 +233,14 @@ void publishMqtt()
 
 bool initMqtt()
 {
+  DEBUG_PRINTLN(F("Initializing MQTT"));
+  // set the important variables
+  mqttEnabled = true;
+  strlcpy(mqttServer, "server address here", MQTT_MAX_SERVER_LEN + 1);
```

Comment:
Surely these connection values are already defined as part of WLED or you wouldn't be able to use it all. No need to add a new compile time value?